src/java.base/share/classes/java/util/UUID.java

```diff
@@ -463,7 +468,54 @@ public long node() {
      */
     @Override
     public String toString() {
-        return jla.fastUUID(leastSigBits, mostSigBits);
+        long lsb = leastSigBits;
+        long msb = mostSigBits;
+        short[] digits = HexDigits.DIGITS;
+        byte[] buf = new byte[36];
+        ByteArray.setLong(
+                buf,
+                0,
+                ((long) digits[((int) (msb >> 56)) & 0xff] << 48)
+                        | ((long) digits[((int) (msb >> 48)) & 0xff] << 32)
+                        | ((long) digits[((int) (msb >> 40)) & 0xff] << 16)
+                        | digits[((int) (msb >> 32)) & 0xff]);
+        buf[8] = '-';
+        ByteArray.setInt(
+                buf,
+                9,
+                (digits[(((int) msb) >> 24) & 0xff] << 16)
+                        | digits[(((int) msb) >> 16) & 0xff]);
+        buf[13] = '-';
+        ByteArray.setInt(
+                buf,
+                14,
+                (digits[(((int) msb) >> 8) & 0xff] << 16)
+                        | digits[((int) msb) & 0xff]);
+        buf[18] = '-';
+        ByteArray.setInt(
+                buf,
+                19,
+                (digits[(((int) (lsb >> 56))) & 0xff] << 16)
+                        | digits[(((int) (lsb >> 48))) & 0xff]);
+        buf[23] = '-';
+        ByteArray.setLong(
+                buf,
+                24,
+                ((long) digits[(((int) (lsb >> 40))) & 0xff] << 48)
+                        | ((long) digits[((int) (lsb >> 32)) & 0xff] << 32)
+                        | ((long) digits[(((int) lsb) >> 24) & 0xff] << 16)
+                        | digits[(((int) lsb) >> 16) & 0xff]);
+        ByteArray.setInt(
+                buf,
+                32,
+                (digits[(((int) lsb) >> 8) & 0xff] << 16)
+                        | digits[((int) lsb) & 0xff]);
+
+        try {
+            return jla.newStringNoRepl(buf, StandardCharsets.ISO_8859_1);
```

Comment:
in java.base it's better to use `ISO_8859_1.INSTANCE` directly instead of `StandardCharsets.ISO_8859_1`.
See [JDK-8234147](https://bugs.openjdk.org/browse/JDK-8234147)

Reply:
done